src/codecs/encoding/config.rs

```diff
@@ -107,10 +107,12 @@ impl EncodingConfigWithFraming {
             (None, Serializer::Avro(_) | Serializer::Native(_)) => {
                 LengthDelimitedEncoder::new().into()
             }
+            (None, Serializer::Gelf(_)) => { 
+                CharacterDelimitedEncoder::new(0).into() 
```

Comment:
```suggestion
                // <add a note here>
                CharacterDelimitedEncoder::new(0).into() 
```

Reply:
just added a commit with a comment linking to the graylog GitHub issue.

Reply:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏just noticed that is using doc comment notation. We generally want to reserve that for docs that are part of user facing documentation. 


```suggestion
                // Graylog/GELF always uses null byte delimiter on TCP, see
                // https://github.com/Graylog2/graylog2-server/issues/1240
                CharacterDelimitedEncoder::new(0).into() 
```

Reply:
I noticed that, too... but I am a complete Rust n00b, so I just did what those around me did (best excuse ever). Will fix it for my comment.